src/components/LoanPositionsTable/ApyCell.tsx

```diff
@@ -28,12 +29,13 @@ const ApyCell = ({
   earnedInterest,
   prices,
   isBorrow = false,
+  negativeApy = false,
   onClick
 }: ApyCellProps): JSX.Element => {
   const rewardsApy = getSubsidyRewardApy(currency, rewards, prices);
 
   const totalApy = isBorrow ? apy.sub(rewardsApy || 0) : apy.add(rewardsApy || 0);
-  const totalApyLabel = getApyLabel(totalApy);
+  const totalApyLabel = negativeApy ? `- ${getApyLabel(totalApy)}` : getApyLabel(totalApy);
```

Comment:
You also need to change `BreakdownGroup.tsx`, which is the tooltip rendered when this label is hovered.

Reply:
Yeah, I wondered the same and posted a question in the OG issue. 

I didn't go ahead with adding negative as the title said "Borrow". Anyway, I added it now, thanks for confirming